frontend/main.js

```diff
@@ -1,13 +1,74 @@
 const { app, BrowserWindow, dialog, ipcMain } = require("electron/main");
 const path = require("node:path");
-
+const fs = require("fs-extra");
+const { PythonShell } = require("python-shell");
+const constants = require("./constants/constants.js");
 let mainWindow;
 
+function runProgram(filePath) {
+  mainWindow.loadFile("loading.html");
+
+  const script = constants.SCRIPT;
+
+  const poetryArgs = [
+    constants.COMMANDS.DEDUPE_DATA,
```

Comment:
It looks like this is only running the test data and not the uploaded data. Looks great otherwise

Reply:
Nevermind

Reply:
By this I mean that it is using the `TEST` format and therefore will only run on test data.